header.php

```diff
@@ -33,9 +33,11 @@
 
 <?php
 if ( ! function_exists( 'elementor_theme_do_location' ) || ! elementor_theme_do_location( 'header' ) ) {
-	if ( did_action( 'elementor/loaded' ) && hello_header_footer_experiment_active() ) {
-		get_template_part( 'template-parts/dynamic-header' );
-	} else {
-		get_template_part( 'template-parts/header' );
+	if ( apply_filters( 'hello_elementor_header_footer', true ) ) {
```

Comment:
use `not (!)` condition and exit from the function

Reply:
It's not a function...